src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx

```diff
@@ -22,76 +21,62 @@ import { AssetRenderer } from '../../../../components/AssetRenderer';
 import { TxFeeCalculator } from '../../../MarginTradePage/components/TxFeeCalculator';
 import { DialogButton } from 'app/components/Form/DialogButton';
 import { TxDialog } from '../../../../components/Dialogs/TxDialog';
-import { usePoolToken } from '../../../../hooks/amm/usePoolToken';
 import { LoadableValue } from '../../../../components/LoadableValue';
-import {
-  LiquidityPool,
-  LiquidityPoolSupplyAsset,
-} from '../../../../../utils/models/liquidity-pool';
 import { weiToNumberFormat } from '../../../../../utils/display-text/format';
 import { useLiquidityMining_getUserInfo } from '../../hooks/useLiquidityMining_getUserInfo';
 import { Asset } from '../../../../../types';
 import { contractReader } from '../../../../../utils/sovryn/contract-reader';
-import { useCacheCallWithValue } from '../../../../hooks/useCacheCallWithValue';
 import { useMining_ApproveAndRemoveLiquidityV1 } from '../../hooks/useMining_ApproveAndRemoveLiquidityV1';
 import { useLiquidityMining_getUserAccumulatedReward } from '../../hooks/useLiquidityMining_getUserAccumulatedReward';
 import { useMaintenance } from 'app/hooks/useMaintenance';
 import { ErrorBadge } from 'app/components/Form/ErrorBadge';
 import { discordInvite } from 'utils/classifiers';
+import { AmmLiquidityPool } from 'utils/models/amm-liquidity-pool';
+
+import { useCacheCallToWithValue } from 'app/hooks/chain/useCacheCallToWithValue';
+import erc20Abi from 'utils/blockchain/abi/erc20.json';
```

Comment:
Can't we just add it to `contracts` and then use it like this where needed `getContract('erc20').abi`? I know there is no actual contract here so we don't have an address but we can just add some placeholder address because we are importing this abi on many places.